packages/zowe-explorer/CHANGELOG.md

```diff
@@ -31,7 +33,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Removed extra files from the VSIX bundle to reduce download size by 64%. [#2042](https://github.com/zowe/vscode-extension-for-zowe/pull/2042)
 - Surfaced any errors from a dataset Recall/Migrate operation. [#2032](https://github.com/zowe/vscode-extension-for-zowe/issues/2032)
 - Re-implemented regular dataset API call if the dataSetsMatching does not exist. [#2084](https://github.com/zowe/vscode-extension-for-zowe/issues/2084)
-- Prevented creation of duplicate session after executing a favorited search query. [#1029](https://github.com/zowe/vscode-extension-for-zowe/issues/1029)
+- Removed excess pop-ups when listing/opening USS files, and replaced required pop-ups with status bar items to improve UX. [#2091](https://github.com/zowe/vscode-extension-for-zowe/issues/2091)
```

Comment:
Line 36 should be above `2.6.0`, but I accidentally put it in the `2.6.0` section for my previous PR - sorry about that! It should be up where your changelog comment is.

Reply:
Should be resolved now